File: 20220831/모범답안/20220831_04.js/DisasterAssistanceAmount.py
import math
from functools import reduce


def solution(arr):
    """
    :param arr: int[]
    :return: int
    """

    arr.sort()

    # 최소, 최대값 제거
    arr = arr[1:-1]

    average = reduce(lambda pre, cur: pre + cur, arr, 0) / len(arr)
    return math.floor(average)

File: 20220831/모범답안/20220831_04.js/DisasterAssistanceAmount.js
/**
 * @param arr {number[]}
 * @returns {number}
 */
function solution(arr) {
  arr.sort((a, b) => a - b)

  // 최소값 제거
  arr.shift()

  // 최대값 제거
  arr.pop()

  const average = arr.reduce((pre, cur) => pre + cur, 0) / arr.length
  return Math.floor(average)
}

solution
